### PR TITLE
Add specific classes to `filterGroup` and `filterItem` of the `filter…

### PR DESCRIPTION
**Pull request by leoWorkingGood**
#308 ## What is the purpose of this pull request?

The purpose of this PR is to customize filters distinctly adding a class to them as there is on `desktop filters`

#### What problem is this solving?

It's solving the issue on mobile filters, that we can't apply different CSS modifiers to each filter group

#### How should this be manually tested?

Here it is an workspace for further tests https://leo2--worldwidegolf.myvtex.com/mens-spiked-golf-shoes

And here it is a print showing this changes https://prnt.sc/sg7hze

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [*] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/AccordionFilterGroup.js [hunk 1]
```diff
@@ -16,10 +21,12 @@ const AccordionFilterGroup = ({
   onOpen,
   onFilterCheck,
 }) => {
+  const handles = useCssHandles(CSS_HANDLES)
   const filters = useSelectedFilters(facets)
   const quantitySelected = filters.filter(facet => facet.selected).length
   const intl = useIntl()
   const facetTitle = getFilterTitle(title, intl)
+  const slugifyFaceTitle = searchSlugify(facetTitle)
```
Comment (iaronaraujo) [inline react/components/AccordionFilterGroup.js:29]:
```suggestion
  const slugifiedFacetTitle = searchSlugify(facetTitle)
```

File: react/components/AccordionFilterGroup.js [hunk 2]
```diff
@@ -1,11 +1,16 @@
 import React from 'react'
 import { useIntl } from 'react-intl'
+import classNames from 'classnames'
```
Comment (iaronaraujo) [inline react/components/AccordionFilterGroup.js:3]: Is this import being used?

File: react/components/AccordionFilterGroup.js (hunk 1 above)
Comment (victorhmp) [inline react/components/AccordionFilterGroup.js:29]:
```suggestion
  const slugifyFacetTitle = searchSlugify(facetTitle)
```

File: react/components/AccordionFilterGroup.js [hunk 3]
```diff
@@ -1,11 +1,16 @@
...
+import { useCssHandles, applyModifiers } from 'vtex.css-handles'
 
 import AccordionFilterItem from './AccordionFilterItem'
 import FacetCheckboxList from './FacetCheckboxList'
 import useSelectedFilters from '../hooks/useSelectedFilters'
 
 import { getFilterTitle } from '../constants/SearchHelpers'
+import { searchSlugify } from '../utils/slug'
+
+const CSS_HANDLES = ['accordionFilterOpened']
```
Comment (victorhmp) [inline react/components/AccordionFilterGroup.js:13]:
```suggestion
const CSS_HANDLES = ['accordionFilterOpen']
```

File: react/components/AccordionFilterGroup.js [hunk 4]
```diff
@@ -29,7 +36,10 @@ const AccordionFilterGroup = ({
       onOpen={onOpen}
       quantitySelected={quantitySelected}
     >
-      <div className={className}>
+      <div className={classNames(
+          applyModifiers(handles.accordionFilterOpened, slugifyFaceTitle),
```
Comment (victorhmp) [inline react/components/AccordionFilterGroup.js:40]:
```suggestion
          applyModifiers(handles.accordionFilterOpen, slugifyFaceTitle),
```

File: react/components/FacetCheckboxList.js [hunk 5]
```diff
@@ -1,17 +1,21 @@
...
+import { applyModifiers } from 'vtex.css-handles'
 
 import styles from '../searchResult.css'
 
+import { searchSlugify } from '../utils/slug'
+
 const FacetCheckboxList = ({ facets, onFilterCheck, facetTitle }) => {
   return facets.map(facet => {
     const { name } = facet
+    const slugifyName = searchSlugify(name)
```
Comment (iaronaraujo) [inline react/components/FacetCheckboxList.js:13]:
```suggestion
    const slugifiedName = searchSlugify(name)
```

File: react/components/AccordionFilterGroup.js (hunk 2 above)
Comment (leoWorkingGood) [reply react/components/AccordionFilterGroup.js:3]: Yes, it is being used on line 40. 
You can check it by seeing the whole file

**Comment by leoWorkingGood** [comment]
Hello @iaronaraujo I updated, to all changes you requested

**Comment by leoWorkingGood** [comment]
@victorhmp I did a `rebase` with `master` but it didn't seem to have any changes?

Could you check, please?

**Comment by victorhmp** [comment]
@leoWorkingGood Sure! The rebase was to resolve this conflict in the CHANGELOG file.
No problem, I'll resolve it and we can release this on Monday :)

**Comment by victorhmp** [comment]
@leoWorkingGood apparently I don't have permission to push to your branch :(
Anyway, I'll leave a suggestion in the CHANGELOG that should enable Git to solve this conflict by itself.

File: CHANGELOG.md [hunk 6]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- Add specific classes to `filterGroup` and `filterItem` of the `filterAccordion` (filter mobile)
+
```
Comment (victorhmp) [inline CHANGELOG.md:12]:
```suggestion
## [Unreleased]
### Added
- Add specific classes to `filterGroup` and `filterItem` of the `filterAccordion` (filter mobile)

```

Comment (victorhmp) [reply CHANGELOG.md:12]: I know it looks weird but removing this line should enable Git to merge this without conflicts :)

**Comment by leoWorkingGood** [comment]
@victorhmp commited the changed, everything looks good.

**Pull request merged**